packages/app/src/elements/settings-security.ts

```diff
@@ -657,30 +659,24 @@ export class SettingsSecurity extends StateMixin(Routing(LitElement)) {
                                           : $l("never")}
                                   </div>
                               `
-                            : until(
-                                  supportsPlatformAuth.then((supported) =>
-                                      supported
-                                          ? html`
-                                                <div class="tag" title="Not supported on this device.">
-                                                    <pl-icon icon="check" class="inline"></pl-icon> ${$l("Supported")}
-                                                </div>
-                                            `
-                                          : html`
-                                                <div class="tag" title="Not supported on this device.">
-                                                    <pl-icon icon="forbidden" class="inline"></pl-icon> ${$l(
-                                                        "Not Supported"
-                                                    )}
-                                                </div>
-                                            `
-                                  ),
-                                  ""
-                              )}
+                            : supportsPlatformAuth
+                            ? html`
+                                  <div class="tag" title="Supported on this device.">
+                                      <pl-icon icon="check" class="inline"></pl-icon> ${$l("Supported")}
+                                  </div>
+                              `
+                            : html`
+                                  <div class="tag" title="Not supported on this device.">
+                                      <pl-icon icon="forbidden" class="inline"></pl-icon> ${$l("Not Supported")}
+                                  </div>
+                              `}
                     </div>
                 </div>
                 <pl-toggle
                     .active=${live(app.remembersMasterKey)}
                     class="click"
                     @change=${(e: Event) => this._toggleBiometricUnlock(e)}
+                    ?notap=${!isBioMetricUnlockSupported}
```

Comment:
While I found this unnecessary after I made the fixes above, it still had a nice touch to having a visual feedback on the cursor change _anyway_.